docker/pytorch-aarch64/build.sh

```diff
@@ -30,11 +30,12 @@ function print_usage_and_exit {
   echo "      --onednn / --dnnl        Build and link to oneDNN / DNNL:"
   echo "                                 * reference    - use the C++ reference kernels throughout."
   echo "                                 * acl          - use Compute Library (default)."
+  echo "      --xla                    Build and install torch-xla wheel".
   echo "      --build-type             Type of build to perform:"
   echo "                                 * base       - build the basic portion of the image, OS and essential packages"
   echo "                                 * libs       - build image including maths libraries and Python3."
   echo "                                 * tools      - build image including Python3 venv, with numpy."
-  echo "                                 * dev        - build image including Bazel and PyTorch, with sources."
+  echo "                                 * dev        - build image including Bazel, PyTorch, PyTorch XLA (if selected) with sources."
   echo "                                 * pytorch    - build image including PyTorch build and benchmarks installed"
```

Comment:
If `--xla` is set, the XLA build will be present in the PyTorch and Full images - this isn't clear from the explanation here.

Reply:
sure, will add the same note in Full image section as well